.github/workflows/build.yml

```diff
@@ -1126,13 +1126,14 @@ jobs:
         runs-on: ${{ matrix.os }}
         needs: [initialize]
         env:
-            PYODIDE_VERSION: 0.23.2
+            # NOTA BENE: When upgrading past 0.24.1, remove workaround below for https://github.com/pyodide/pyodide/issues/4216
+            PYODIDE_VERSION: 0.24.1
             # PYTHON_VERSION and EMSCRIPTEN_VERSION are determined by PYODIDE_VERSION.
             # The appropriate versions can be found in the Pyodide repodata.json
             # "info" field, or in Makefile.envs:
             # https://github.com/pyodide/pyodide/blob/main/Makefile.envs#L2
-            PYTHON_VERSION: 3.11.2
-            EMSCRIPTEN_VERSION: 3.1.32
+            PYTHON_VERSION: 3.11.3
```

Comment:
this also changed? Is this durable to changes in `setup-python`?

Reply:
ah meh ` # PYTHON_VERSION and EMSCRIPTEN_VERSION are determined by PYODIDE_VERSION.`

Reply:
all gucci?